okhttp/src/main/java/okhttp3/internal/http/HttpEngine.java

```diff
@@ -131,50 +131,20 @@
    */
   public final boolean bufferRequestBody;
 
-  /**
-   * The original application-provided request. Never modified by OkHttp. When follow-up requests
-   * are necessary, they are derived from this request.
-   */
-  private final Request userRequest;
```

Comment:
Cool - I noted the class level javadocs still reference sendRequest and readResponse.